services/exposurelog/templates/deployment.yaml

```diff
@@ -56,18 +56,18 @@ spec:
             - name: BUTLER_URI_2
               value: {{ .Values.config.butler_uri_2 | quote }}
             - name: EXPOSURELOG_DB_USER
-              value: exposurelog
+              value: {{ .Values.db.user | quote }}
             - name: EXPOSURELOG_DB_PASSWORD
               valueFrom:
                 secretKeyRef:
                   name: postgres
                   key: exposurelog_password
-            - name: EXPOSURELOG_DB_HOST
-              value: postgres.postgres
-            - name: EXPOSURELOG_DB_PORT
-              value: "5432"
-            - name: EXPOSURELOG_DB_DATABSE
-              value: exposurelog
+            - name: NARRATIVELOG_DB_HOST
+              value: {{ .Values.db.host | quote }}
+            - name: NARRATIVELOG_DB_PORT
+              value: {{ .Values.db.port | quote }}
+            - name: NARRATIVELOG_DB_DATABSE
+              value: {{ .Values.db.database | quote }}
```

Comment:
The names here are different; was that intentional?

Reply:
I don't know why Russell left out the last `A` in Database for the env var.  So, yeah, because I couldn't bring myself to put a `databse` configuration setting.

Reply:
Derp.  That musta been on me.  As was all the confusion between exposure and narrativelog.